src/components/Sidebar/components/NavItem/NavItem.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'react-emotion';
+
+const baseStyles = ({ theme }) => css`
+  label: nav-item;
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  height: auto;
+  margin: ${theme.spacings.mega};
+  padding: ${theme.spacings.bit};
+  cursor: pointer;
+  color: ${theme.colors.n500};
+  fill: ${theme.colors.n500};
+`;
+
+const hoverStyles = ({ theme, selected }) => css`
+  label: nav-item__hover;
+  @media (hover: hover) {
+    &:hover {
+      color: ${!selected && theme.colors.n300};
+      fill: ${!selected && theme.colors.n300};
+    }
+  }
+`;
+
+const activeStyles = ({ theme, selected }) => css`
+  label: nav-item__active;
+  color: ${selected && theme.colors.n100};
+  font-weight: ${selected && theme.fontWeight.bold};
```

Comment:
I'm fairly certain that this produces invalid CSS if `selected` is falsy. By convention, we make the whole group of styles conditional: 

```js
const activeStyles = ({ theme, selected }) => selected && css`
  label: nav-item__active;
  color: ${theme.colors.n100};
  font-weight: ${theme.fontWeight.bold};
`;